src/timeline-list/index.tsx

```diff
@@ -53,13 +53,14 @@ export interface TimelineListProps {
 
 const TimelineList = (props: TimelineListProps) => {
   const {timelineProps, events, renderItem, showNowIndicator, scrollToFirst, scrollToNow, initialTime} = props;
+  const shouldFixRTL = constants.isRTL && (constants.isRN73() || constants.isAndroid); // isHorizontal = true
```

Comment:
consider deleting commented out code

Reply:
This comment comes to hit that we assume the list is in horizontal layout since the TimelineList only supports horizontal